src/components/TopBar/TopBar.js

```diff
@@ -766,20 +765,25 @@ TopBar.propTypes = {
     appName: PropTypes.string,
     appColor: PropTypes.string,
     appLogo: PropTypes.object,
+    appVersion: PropTypes.string,
+    appLicense: PropTypes.string,
     user: PropTypes.object,
     children: PropTypes.node,
     appsAndUrls: PropTypes.array,
     onThemeClick: PropTypes.func,
     theme: PropTypes.string,
     onAboutClick: PropTypes.func,
+    getGlobalVersion: PropTypes.func,
+    getAdditionalComponents: PropTypes.func,
     onEquipmentLabellingClick: PropTypes.func,
     equipmentLabelling: PropTypes.bool,
     withElementsSearch: PropTypes.bool,
     searchDisabled: PropTypes.bool,
     searchingLabel: PropTypes.string,
-    onSearchTermChange: PropTypes.func,
-    onSelectionChange: PropTypes.func,
-    elementsFound: PropTypes.array,
+    onSearchTermChange: PropTypes.func.isRequired,
+    onSelectionChange: PropTypes.func.isRequired,
+    elementsFound: PropTypes.array.isRequired,
```

Comment:
Why is it required now ? For example in GridExplore we have a TopBar but no search functionality

Reply:
that for another PR, sorry